com.unity.ml-agents/Tests/Editor/MLAgentsEditModeTest.cs

```diff
@@ -263,11 +263,16 @@ public void TestAcademy()
         public void TestAcademyAutostep()
         {
             var aca = Academy.Instance;
-            Assert.IsTrue(aca.IsAutomaticSteppingEnabled);
+            Assert.IsTrue(aca.AutomaticSteppingEnabled);
+
+            // Note that calling
```

Comment:
@surfnerd Thoughts on this? I assume Object.Destroy() is generally preferable, but unit tests break if we do that instead of Object.DestroyImmediate()

Reply:
Seems good since it's `false` by default.